pay-api/src/pay_api/services/refund.py

```diff
@@ -216,7 +216,10 @@ def create_routing_slip_refund(cls, routing_slip_number: str, request: Dict[str,
         # Rejected refund makes routing slip active
         if refund_status == RoutingSlipStatus.REFUND_REJECTED.value:
```

Comment:
rename this to `status == RoutingSlipStatus.REFUND_REJECTED.value`

so it's not confusing with the true `refund_status`